README.md

```diff
@@ -1,135 +1,61 @@
-# Auction system
+[![Lagom](docs/logo.png)](https://github.com/lagom/lagom)
 
```

Comment:
logo looks great but too big IMHO.

Please resize to half or less. See https://gist.github.com/uupaa/f77d2bcf4dc7a294d109